app/controllers/content_items_controller.rb

```diff
@@ -136,16 +136,41 @@ def set_guide_draft_access_token
     @content_item.draft_access_token = params[:token]
   end
 
+  class GdsApi::Response
+    def update_response_body
```

Comment:
+1 for making another change to the API adapters to have them do the `dig`ging and keys-transforming for us, if that would enable us to avoid this monkey patch?

Reply:
That's a good idea.  I'll look into this.